modules/react/loading-dots/stories/examples/Accessible.tsx

```diff
@@ -11,8 +11,13 @@ const styleOverrides = {
     gap: system.space.x4,
   }),
   loadingStyles: createStyles({
-    backgroundColor: system.color.bg.muted.default,
-    padding: system.space.x3,
+    borderRadius: system.shape.round,
+    backgroundColor: system.color.bg.overlay,
+    height: 80,
+    width: 80,
+    alignItems: 'center',
+    justifyContent: 'center',
+    display: 'flex',
```

Comment:
It looks like you're using this example to change the backgroundColor, would you be oppose to adding a storybook example like `Custom Color` or `Custom Background Color` to highlight it and then we can add it to the docs?

Reply:
Yeah you know what, it might be worth breaking this out into a "Circle Variant" story to match https://canvas.workdaydesign.com/components/indicators/loading-dots/ better - I've been thinking about it solely for accessibility on dark backgrounds but you might want it without all the aria-live stuff included here. Good shout!